examples/data-objects/todo/tsconfig.json

```diff
@@ -1,9 +1,7 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
 	"compilerOptions": {
-		"module": "esnext",
-		"outDir": "lib",
-		"jsx": "react",
+		"outDir": "./lib",
 		"types": [
 			"react",
 			"react-dom",
```

Comment:
It is not a good practice to build product code with test code. Test types are included below vvvv.
Although we don't have anything using it, I think this is meant to exist as an example library. Assuming not simple split now we can file a follow-up debt item.
(blocking just for visibility)